### PR TITLE
[SW-2119] Make SW Compatible with Older Versions of Steam

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Comment by mn-mikke** [comment]
@jakubhava  the problem is not with the current version of steam, but due to this change in the stracture the older versions of steam are not able run newer SW versions. Not all customers can upgrade steam easily.

File: dist/build.gradle [hunk 1]
```diff
@@ -174,6 +175,10 @@ task copyFilesForZipDistribution {
         copyRecursive("bin/", "$zipDir/bin")
         copyRecursive("doc/build/site/", "$zipDir/doc/build/site")
         copyRecursive("docker/", "$zipDir/docker")
+
+        // TODO: The below lines should be removed in future. For now, they ensure compatibility with older versions of Steam.
```
Comment (jakubhava) [inline dist/build.gradle]: @mn-mikke please create JIRA fro removal

**Comment by jakubhava** [comment]
Yup, realized it's on older versions -> approved

File: dist/build.gradle (hunk 1 above)
Comment (mn-mikke) [reply dist/build.gradle]: https://0xdata.atlassian.net/browse/SW-2121

**Comment by bilcus** [comment]
> I would prefer if Steam handles this -> it should be matter of checking whether the file exists on the given path right? @bilcus

I will change Steam to be more flexible about the location of the files.

**Comment by jakubhava** [comment]
Thank you @bilcus!

**Pull request merged**